<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bring_your_own_png</class>
 <widget class="QDialog" name="bring_your_own_png">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bring Your Own PNG</string>
  </property>
  <widget class="QGraphicsView" name="graphic_original">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Click to load a picture</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sceneRect">
    <rectf>
     <x>0.000000000000000</x>
     <y>0.000000000000000</y>
     <width>100.000000000000000</width>
     <height>100.000000000000000</height>
    </rectf>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="rubberBandSelectionMode">
    <enum>Qt::IntersectsItemShape</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphic_prepared">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use median filter and contrast to see a difference here</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sceneRect">
    <rectf>
     <x>0.000000000000000</x>
     <y>0.000000000000000</y>
     <width>100.000000000000000</width>
     <height>100.000000000000000</height>
    </rectf>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphic_svg_eu">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Click on &quot;Try it&quot; to see your vector graphic here</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sceneRect">
    <rectf>
     <x>0.000000000000000</x>
     <y>0.000000000000000</y>
     <width>100.000000000000000</width>
     <height>100.000000000000000</height>
    </rectf>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_preparation">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gaze</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Preparation</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_autotrace">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gaze</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Autotrace</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_arrow">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Arrow.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_autotrace">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gaze</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="toolTip">
    <string>Autotrace your picture to see the result</string>
   </property>
   <property name="text">
    <string>Try it</string>
   </property>
  </widget>
  <widget class="QSlider" name="prepa_slider_medianfilter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTip">
    <string>Set the median filter to higher values to remove noise, or to lower values to preserve detail.</string>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="sliderPosition">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="prepa_label_medianfilter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gaze</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Median Filter</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="prepa_line_edit_medianfilter">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set the median filter to higher values to remove noise, or to lower values to preserve detail.</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="prepa_label_contrast">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gaze</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>To make your symbol really black and white, enhance contrast here</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Contrast Enhancing</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSlider" name="prepa_slider_contrast">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>To make your symbol really black and white, enhance contrast here</string>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
   <property name="sliderPosition">
    <number>20</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
   <property name="tickInterval">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="prepa_line_edit_contrast">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>To make your symbol really black and white, enhance contrast here</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_next">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>next step</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QSlider" name="prepa_slider_despeckle">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>240</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set the mid point of the final color palette</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>55</number>
   </property>
   <property name="sliderPosition">
    <number>55</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="prepa_label_middle_color">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>220</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gaze</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Set the mid point of the final color palette</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Middle color</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="prepa_line_edit_despeckle">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>240</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set the mid point of the final color palette</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="text">
    <string>55</string>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QPushButton" name="button_upsampling">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gaze</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Upsampling</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="prepa_line_edit_errorthreshold">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>290</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Accepts stranges lines on high values, deletes them on low values</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QSlider" name="prepa_slider_errorthreshold">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>290</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Accepts stranges lines on high values, deletes them on low values</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="sliderPosition">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="prepa_label_errorthreshold">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>270</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gaze</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Accepts stranges lines on high values, deletes them on low values</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Error Threshold</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSlider" name="prepa_slider_corner">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>340</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>How many pixels make a corner? More -&gt; rounder Less -&gt; mor corners</string>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
   <property name="sliderPosition">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="prepa_line_edit_corner">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>340</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>How many pixels make a corner? More -&gt; rounder Less -&gt; mor corners</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="prepa_label_corner">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>320</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gaze</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>How many pixels make a corner? More -&gt; rounder Less -&gt; mor corners</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Corner Detect</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_filename">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="prepa_label_new_filename">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gaze</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>New Filename</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_cancel</sender>
   <signal>clicked()</signal>
   <receiver>bring_your_own_png</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_next</sender>
   <signal>clicked()</signal>
   <receiver>bring_your_own_png</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
